root/collection/index.tt

```diff
@@ -19,6 +19,9 @@
       [%- IF collection.type.item_entity_type == 'event' -%]
         [% React.embed(c, 'components/EventsList', { events => entities, checkboxes => my_collection ? 'remove' : '', order => order, sortable => 1})
             WRAPPER 'components/with-pager.tt' %]
+      [%- ELSIF collection.type.item_entity_type == 'artist' -%]
+        [% React.embed(c, 'components/ArtistsList', { artists => entities, checkboxes => my_collection ? 'remove' : '', noAreas => 1, order => order, sortable => 1})
```

Comment:
Hiding areas on collection page since the three area columns might be a bit overwhelming and nobody has asked for those so far :)